Packs/XSOARmirroring/Integrations/XSOARmirroring/XSOARmirroring.py

```diff
@@ -609,14 +615,26 @@ def get_remote_data_command(client: Client, args: Dict[str, Any], params: Dict[s
 
         demisto_debug(f'tags: {tags}')
         demisto_debug(f'tags: {categories}')
+        integration_context = get_integration_context()
+        XSOARMirror_mirror_reset: dict = json.loads(integration_context.get('XSOARMirror_mirror_reset', None))
```

Comment:
2 notes:
1. json.loads(None) will return an error, either do it like this or wrap it with a condition.
2. Please use lower case for `XSOARMirror_mirror_reset` name.
```suggestion
        XSOARMirror_mirror_reset: dict = json.loads(integration_context.get('XSOARMirror_mirror_reset', '{}'))
```

Reply:
done.